src/exposure-provider.tsx

```diff
@@ -319,12 +333,9 @@ export const ExposureProvider: React.FC<ExposureProviderProps> = ({
 
   const getCloseContacts = async () => {
     try {
-      if (state.permissions.exposure.status === PermissionStatus.Allowed) {
-        const contacts = await ExposureNotification.getCloseContacts();
-        setState((s) => ({...s, contacts}));
-        return contacts;
-      }
-      return [];
+      const contacts = await ExposureNotification.getCloseContacts();
```

Comment:
Is this really safe to remove the permission check?

Reply:
Further - if known by the provider that perms are not allowed - we could save the fn call with the previous implementation which then does not have to go over the native bridge etc

Reply:
Yes, reading contacts does not involve ENS

Reply:
In either case - if it is known perms are not granted - therefore there will be no contacts - the check is better than the call

Reply:
Perms can be removed by making another app active but this app can still have contacts